src/runtime/composables/useAsyncAlgoliaSearch.ts

```diff
@@ -13,7 +13,7 @@ export async function useAsyncAlgoliaSearch ({ query, requestOptions, indexName
 
   const algoliaIndex = useAlgoliaInitIndex(index)
 
-  const result = await useAsyncData(`${index}-async-search-result`, async () => {
+  const result = await useAsyncData(`${index}-async-search-result-${key ?? ''}`, async () => {
```

Comment:
issue: I think this `key` is not destructured from the `SearchParams` in the `useAsyncAlgoliaSearch` composable as is for example `indexName`

Reply:
Yeah you qre right, I did it in a hurry via the github UI, give me ten minutes and I'll fix it 😅❤️